contrib/premake/zstd.lua

```diff
@@ -0,0 +1,79 @@
+-- This GENie/premake file copies the behavior of the Makefile in the lib folder.
+-- Basic usage: project_zstd(ZSTD_DIR)
+
+function project_zstd(dir, compression, decompression, deprecated, dictbuilder, legacy)
+	if compression == nil then compression = true end
+	if decompression == nil then decompression = true end
+	if deprecated == nil then deprecated = false end
+	if dictbuilder == nil then dictbuilder = false end
+
+	if legacy == nil then legacy = 0 end
+
+	if compression then
+		dictbuilder = false
```

Comment:
Why turn `dictBuilder` and `deprecated` off?

Reply:
Ugh, nevermind, I think I was tired writing this. I got these wrong too, you're right.